website/src/releases/hostname-migration.mdx

```diff
@@ -0,0 +1,70 @@
+---
+date: 2020-06-10
+title: Migrate Custom Applications to the new host naming scheme
+description: |
+  The Merchant Center is adopting a new host naming scheme and deprecating the old hostnames.\
+  Therefore, Custom Applications must be updated to support both hostnames, for backwards compatibility.
+type: announcement
+topics:
+  - Merchant Center
+  - Configuration
+---
+
+Recently we [introduced a new host naming scheme](/main-concepts/api-gateway#cloud-regions) for the commercetools APIs, including the Merchant Center, to increase region and cloud provider transparency.
+
+While the previous hostnames such as `*.sphere.io` (Google Cloud, Belgium), `*.commercetools.com` (Google Cloud, Belgium), and `*.commercetools.co` (Google Cloud, Iowa) continue to be operational, we **strongly recommend** to migrate to the new hostname structure.
+
+In regards to Custom Applications, the [runtime configuration](/deployment/runtime-configuration) needs to be updated to **support both the new and the legacy hostnames**, to allow users to log in with the new Merchant Center hostname and to be backwards compatible with the legacy hostnames.
+
+Here are the steps to take.
+
+<!--more-->
+
+# 1. Update the `env.json`
+
+Change the `mcApiUrl` in the [`env.json`](/deployment/runtime-configuration#envjson) to use the new hostname.
+
+For example:
+
+```json title="Changing the hostname for GCP EU" highlightLines="4"
+{
+  "applicationName": "merchant-center-application-template-starter",
+  "frontendHost": "localhost:3001",
+  "mcApiUrl": "https://mc-api.europe-west1.gcp.commercetools.com",
+  "location": "eu",
+  "env": "development",
+  "cdnUrl": "http://localhost:3001",
+  "servedByProxy": false
+}
+```
+
+<Info>
+
+In production, the `<ApplicationShell>` takes care of inferring the correct [Merchant Center Gateway API hostname](/main-concepts/api-gateway#hostnames) based on the Merchant Center hostname that the user is logged into.
+This results in the Custom Application to work in both new and legacy hostnames, even though the `mcApiUrl` is configured to a specific value.
+
+</Info>
+
+# 2. Update the `headers.json`
+
+Update the [Content Security Policy](/deployment/runtime-configuration#content-security-policy) to allow both new and legacy hostnames.
+
+For example:
+
+```json title="Updating the Content Security Policy for GCP EU" highlightLines="6-7"
+{
+  "csp": {
+    "script-src": ["my-app.now.sh"],
+    "connect-src": [
+      "my-app.now.sh",
+      "mc-api.europe-west1.gcp.commercetools.com",
+      "mc-api.commercetools.com"
+    ],
+    "style-src": ["my-app.now.sh"]
+  }
+}
+```
+
+# 3. Re-deploy the Custom Application
+
+That's it, re-deploy the Custom Application with the updated configuration. Users can now access the Custom Application using both new and legacy hostnames (for backwards compatibility).
```

Comment:
` That's it, re-deploy`.....The comma should be replaced with a semicolon or a period (preferably a period in this case).